interceptors/client_test.go

```diff
@@ -50,7 +50,7 @@ func (m *mockReportable) Equal(t *testing.T, expected []*mockReport) {
 				require.NoError(t, err)
 				continue
 			}
-			require.Equal(t, expected[i].postCalls[k].Error(), err.Error(), "%v %v", i, k)
+			require.EqualError(t, err, expected[i].postCalls[k].Error(), "%v %v", i, k)
```

Comment:
If there is a bug and `err` is `nil`, then this line panics. Now it does not and fails the test cleanly.

Reply:
👍🏽 Thanks a lot